### ISSUE TITLE
Execution failed for task ':compileJava'.

### ISSUE THREAD

**Issue by Potochnyy** [Fabricators-of-Create/Create#25]
> Configure project :
Fabric Loom: 0.10.64
:remapping minecraft (TinyRemapper, official -> named)
:remapping minecraft (TinyRemapper, official -> intermediary)
Processing file: minecraft-project-@-mapped.jar
Processing file: minecraft-project-@-mapped.jar
:remapping 1 mods (TinyRemapper, intermediary -> named)
:remapping 54 mods (TinyRemapper, intermediary -> named)
:remapping 1 mods (TinyRemapper, intermediary -> named)
:remapping 1 mods (TinyRemapper, intermediary -> named)
:remapping sources
Loom is applying dependency data manually to publications instead of using a software component (from(components["java"])). This is deprecated and will be removed in Loom 0.12.

> Task :compileJava FAILED
Deprecated Loom APIs were used in this build, making it incompatible with future versions of Loom. Use Gradle warning modes to control the verbosity of the warnings.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> error: release version 17 not supported

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

**Comment by TropheusJ** [comment]
update your java.
beware:
The mod is not ready for use yet, and we're not providing support until it's ready. Once you fix this issue, it'll then crash from flywheel not being found because we're using a custom private flywheel build temporarily. Even if this wasn't the case, the mod would then crash in production.
tldr: you can't use it.
